test/specs/mini-tooltip.spec.js

```diff
@@ -166,5 +166,23 @@ define(['jquery', 'd3', 'mini-tooltip'], function($, d3, tooltip) {
                 expect(newNameLabel).toBe(testNameLabel);
             });
         });
+
+        it('should allow overriding the value formatting function', () => {
+            let expected = '8',
+                actual;
+
+            tooltipChart.valueFormatter(value => value.toString().length.toString());
```

Comment:
This line could work great for that example

Reply:
I used the line for the `@example` annotation. Please tell me if I should add anything more.